v1.0.0.md

```diff
@@ -15,20 +15,35 @@ Harvester can be installed from
+* Bring your own SSL certificate for the Harvester UI/API. [[DOC](https://docs.harvesterhci.io/v1.0/settings/settings/#ssl-certificates)]
+* Add proxy support to access external services, including the download of images and backup to s3 services. [[DOC](https://docs.harvesterhci.io/v1.0/settings/settings/#http-proxy)]
+* Air gap environment support of both Harvester single cluster and Rancher Harvester node driver support.
+* Allowing Backup and restore VM metadata from the backup target on both existing and new clusters. [[DOC](https://docs.harvesterhci.io/v1.0/vm/backup-restore/)]
+* Harvester RKE1 node driver is now GA and added cloud provider integration. [[DOC](https://docs.harvesterhci.io/v1.0/rancher/rke1-cluster/)]
+* Enhanced Harvester cluster import experience to Rancher. [[DOC](https://docs.harvesterhci.io/v1.0/rancher/virtualization-management/)]
+* Support enable/disable additional data storage partition. [[DOC](https://docs.harvesterhci.io/v1.0/install/harvester-configuration/#installno_data_partition)]
+* Support volume hot-unplugin. [[DOC](https://docs.harvesterhci.io/v1.0/vm/hotplug-volume/)]
+* Multi-disk support enhancement. [[LINKS](https://github.com/harvester/harvester/issues?q=is%3Aissue+label%3Aarea%2Fnode-disk-manager+milestone%3Av1.0.0+)]
+* Support resource over-commit config. [[DOC](https://docs.harvesterhci.io/v1.0/settings/settings/#overcommit-config)]
 
 # Upgrade
 * One click upgrade will be supported from v1.0.0 to future versions.
-* (Experimental) Manual upgrade process is provided to upgrade from v0.3.0 to v1.0.0.
+* (Experimental) Manual upgrade process is provided to upgrade from v0.3.0 to v1.0.0. [[DOC](https://docs.harvesterhci.io/v1.0/upgrade/)]
+
+# Bugs
+* Cloud config byte limit [[#760](https://github.com/harvester/harvester/issues/760)]
+* Frequent kernel panics occurring during operation. [[#1342](https://github.com/harvester/harvester/issues/1342)]
+* USB Boot hangs indefinitely at `fb0: switching to amdgpudrmfb from EFI VGA`. [[#1225](https://github.com/harvester/harvester/issues/1225)]
+* Harvester installer can't resolve hostnames. [[#1590]](https://github.com/harvester/harvester/issues/1590)
+* Unable to migrate restore-new vm twice. [[#1086](https://github.com/harvester/harvester/issues/1086)]
+* Unable to upload image by file [[#1415](https://github.com/harvester/harvester/issues/1415)]
 
 # Known issues
-- VM with unschedulable disks doesn't show clear warning message (#1695)
-- Power on 3 Harvester nodes in sequence may cause VIP connectivity lost (#1681)
-- Switch kubevip pod in Harvester cluster under DHCP (dnsmasq) the VIP is missing and failed to access Harvester (#1645)
-- `harvester` wasn't set as default storage class inside Rancher provisioned RKE2 cluster (#1629)
-- Deleting a host holding the VIP in the cluster makes the VIP inaccessible (#1398)
+- VM with unschedulable disks doesn't show clear warning message ([#1695](https://github.com/harvester/harvester/issues/1695))
```

Comment:
We need some explanations for each of the issues and what would be the workaround.

Reply:
sure, do u think it's better to add in here or the issue comment.

Reply:
We can point to the workaround in the issue’a comment.